pkg/disttask/importinto/planner.go

```diff
@@ -553,3 +585,8 @@ func getRangeSplitter(
 		regionSplitKeys,
 	)
 }
+
+func writeIngestPlanMetaPath(taskID int64, idx int) string {
+	prefix := path.Join(strconv.FormatInt(taskID, 10), "write-ingest-plan", strconv.Itoa(idx))
```

Comment:
maybe unify the path as `<task-id>/plan/<step-str>/<ordinal>/meta.json`

our step string contains `&` right now, you might need to change it too